### PR TITLE
Feature custom width and height of scanning area

### PR DESCRIPTION
**Pull request by OrangeSofter**
Developing feature #368 in progress

### CODE REVIEW

**Comment by juliansteenbakker** [comment]
Can you please run `flutter format .` and commit changes?

**Comment by OrangeSofter** [comment]
Yes i did it

File: lib/src/qr_code_scanner.dart [hunk 1]
```diff
@@ -334,7 +334,7 @@ class QRViewController {
         await channel.invokeMethod('setDimensions', {
           'width': renderBox.size.width,
           'height': renderBox.size.height,
-          'scanArea': overlay?.cutOutSize ?? 0,
+          'scanArea': overlay?.cutOutWidth ?? 0, //Todo
```
Comment (juliansteenbakker) [inline lib/src/qr_code_scanner.dart:337]: What is this todo comment?

Comment (OrangeSofter) [reply lib/src/qr_code_scanner.dart:337]: This is just for me)
I will finish the feature soon and remove the todo comment

Comment (OrangeSofter) [reply lib/src/qr_code_scanner.dart:337]: This is just for me)
In order not to forget to complete the use of the real scanning area on ios
I will finish the feature soon and remove the todo comment

Comment (OrangeSofter) [reply lib/src/qr_code_scanner.dart]: I did it

**Comment by OrangeSofter** [comment]
Feature #368 done

**Comment by juliansteenbakker** [comment]
First of all thank you for your contribution. I have checked it, however i think it would be best if we can leave the current implementation in place, and add this as an optional feature. Otherwise this would be quite a breaking change. I think the best would be to just leave the cutOutSize in place, and to either accept this, OR the separate scan area width and height.

What do you think about this?

**Comment by juliansteenbakker** [comment]
My fault, i thought this removes all references to `cutOutSize` but it doesn't. Thanks !

**Pull request merged**